README.md

```diff
@@ -52,6 +53,7 @@ Control the Kubernetes cluster like `kubectl apply`, `kubectl delete`, `kubectl
 
 #### Parameters
 
+- `namespace`: *Optional.* The namespace to default to. Defaults to `default`.
```

Comment:
I don't understand which namespace is preferred.  source or params? (params will be preferred.)
I recommend to write the priority of namespace.

Reply:
It's params, and I will change the description as follows:

- `namespace`: *Optional.* The namespace scope. It will override the namespace in other params and source configuration.